tools/git/pre-commit

```diff
@@ -44,19 +49,19 @@ if [ ! -x "$FORMAT" ]; then
 fi
 
 source "$GIT_TOP/tools/autopep8.sh"
-if [ ! -d ${AUTOPEP8_VENV} ]; then
+if [ ! -d "${AUTOPEP8_VENV}" ]; then
     echo "Run \"make autopep8\""
     exit 1
 fi
-source ${AUTOPEP8_VENV}/bin/activate
+source "${AUTOPEP8_VENV}/bin/activate"
 
 # Where to store the patch
 clang_patch_file=$(mktemp -t clang-format.XXXXXXXXXX)
 autopep8_patch_file=$(mktemp -t autopep8.XXXXXXXXXX)
-trap "rm -f $clang_patch_file $autopep8_patch_file" 0 1 2 3 5 15
+trap 'rm -f $clang_patch_file $autopep8_patch_file' 0 1 2 3 15
```

Comment:
This change to teh single quote will no longer expand $clang_patch_file and $autopep8_patch_file. Is that an intentional change?

Reply:
The shellcheck issue is:
```
In tools/git/pre-commit line 61:
trap "rm -f $clang_patch_file $autopep8_patch_file" 0 1 2 3 15
            ^---------------^ SC2064 (warning): Use single quotes, otherwise this expands now rather than when signalled.
                              ^------------------^ SC2064 (warning): Use single quotes, otherwise this expands now rather than when signalled.

For more information:
  https://www.shellcheck.net/wiki/SC2064 -- Use single quotes, otherwise this...
```

In this case, I think that expanding these variable when the trap is called is OK,since their values arenever changed.